refactor: Update help wiki (#31)
ReVanced/revanced-documentation: docs/revanced-reddit/wiki/help/troubleshooting.md

# **Troubleshooting**

## No internet connection in YouTube / YT Music

> If your YT ReVanced appears to not have internet and you recently changed your Google password, remove the Vanced MicroG account from within your device settings and then log back in. Alternatively, reinstall Vanced MicroG.

Reviewer: ```suggestion
Log out of the Vanced MicroG account within your device settings and log back in. Alternatively, reinstall Vanced MicroG.
```

## The player UI on YouTube doesn't go away

> You may have it set to permanently show in **Settings > Accessibility**. Otherwise, this issue occurs randomly and the only fix is to restart the app.

Reviewer: ```suggestion
You may set it to show in **Settings > Accessibility**permanently. Otherwise, this issue occurs randomly, and currently the only fix is to restart the app.
```
